### ISSUE TITLE
<kbd> has too much style

### ISSUE THREAD

**Issue by garrett** [patternfly/patternfly-3#1002]
The `<kbd>` element has semantic meaning and represents what someone is supposed to type.

As PatternFly pulls in Bootstrap, and Bootstrap styles all `kbd` elements by default (which is OK for documentation), it causes a problem when `kbd` is used in applications. This is an accessibility issue, as screen readers depend on using various semantic tags to describe the document.

We ran into this problem on Cockpit, and the results look like so:
![screenshot of kbd in action](https://user-images.githubusercontent.com/3228183/37957336-598f2408-31b6-11e8-92fc-e6e6e32e925e.png)

While it makes some sense to style `kbd > kbd` (that is: specific key presses), it's a bad idea to style all `kbd` elements. If a terminal look is desired, a class should be applied or at least it should be contained in a document class wrapper.

The offending code can be seen in [code.less from Bootstrap](https://github.com/patternfly/patternfly-site/blob/3d38a313065aa2e8ea91b68a3377a0fcb94869b8/library/components/patternfly/less/lib/bootstrap/code.less#L24-L38) (this link has a current version of the file from PatternFly's site).

**Comment by thebigtoona** [comment]
I'll preface this with I'm new here, but it sounds like its just pulling out some styles into a separate class, and then documentation on that element might need to change a little to compensate for the new extra class? I can make a separate class for this in code.less and test it out

**Comment by jeff-phillips-18** [comment]
I'm not sure we want Patternfly to override all bootstrap settings. There is no pattern in patternfly for the `<kbd>` element. Is there any reason the application cannot override this setting w/o involving patternfly?

**Comment by thebigtoona** [comment]
I was able to override them with the class I built in the codepen.  I added the class to the patternfly additions that were built. Its in a separate file named code.less, but I had no issue altering it without touching the codebase. The codepen is using an unaltered patternfly

This is the codepen for reference: 

https://codepen.io/tmorozco/pen/aazBMQ

**Comment by LHinson** [comment]
@garrett were you able to resolve your concern in product?

**Comment by garrett** [comment]
@LHinson: Thanks for the follow up!

We decided to rework the UI that would have used the `kbd` tag. Between having to override styles, poor screen reader support (according to various sources on the Internet), and redoing the UI, we decided to not use the tag.

I'm assuming PF4 probably does not style the `kbd` tag anyway, so I suppose this bug is probably now either fixed or simply irrelevant?

**Comment by LHinson** [comment]
@garrett Yeah, given we moved forward without bootstrap, this shouldn't be a problem moving forward with PF4. I'll close it now. Thanks!
